Note for accessing services across docker containers (#9254)
supabase/supabase: apps/reference/docs/guides/cli/local-development.mdx

</TabItem>
</Tabs>

:::note

> If you are looking to access, for example, your database from an edge function within your local Supabase setup, you will need to replace `localhost` with `host.docker.internal` in any URLs.

Reviewer: ```suggestion
To access the database from an edge function in your local Supabase setup, replace `localhost` with `host.docker.internal`.
```
